src/program/OutfitStudio.cpp

```diff
@@ -2003,6 +2002,15 @@ void OutfitStudioFrame::GetNormalizeBones(std::vector<std::string> *normBones, s
 	}
 }
 
+std::vector<std::string> OutfitStudioFrame::GetSelectedBones() {
+	std::vector<std::string> boneList;
+	wxArrayTreeItemIds selItems;
+	outfitBones->GetSelections(selItems);
+	for (int i = 0; i < selItems.size(); i++)
+		boneList.push_back(outfitBones->GetItemText(selItems[i]));
```

Comment:
Doesn't compile for me on my wxWidgets and MSVC++ (pushing wxString into std::string vector).